core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -266,6 +282,12 @@ private void registerUpcheckRoute(
         .failureHandler(errorHandler);
   }
 
+  protected void registerHealthCheckProcedure(
+      final String name,
+      io.vertx.core.Handler<io.vertx.core.Promise<io.vertx.ext.healthchecks.Status>> procedure) {
```

Comment:
Handler doesn't need to be fully qualified we already have that type imported. Do the other types here Promos and the Status need be fully qualified as well or can we use imports?